### PR TITLE
Complex float

### PR DESCRIPTION
**Pull request by nikhilbhagdikar**
div/log/exp are moved to the new test infra with error bounds added.

### CODE REVIEW

File: lassen/utils.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+from codecs import decode
+import struct
+
```
Comment (rdaly525) [inline lassen/utils.py]: We should be using hwtypes.FPVector in order to construct floating point values and for converting between FP and BitVector.

File: tests/test_complex.py [hunk 2]
```diff
@@ -1,14 +1,16 @@
+from lassen.stdlib.fma import gen_FMA
+from lassen.stdlib.fpops import gen_fdiv, gen_fln, gen_fexp
 from collections import namedtuple
 from lassen.stdlib import gen_FMA, gen_Add32, gen_Sub32
 from lassen.isa import DATAWIDTH
 from hwtypes import BitVector, Bit, SIntVector
 from lassen.sim import gen_pe
+from lassen.utils import float2bfbin, bfbin2float
```
Comment (rdaly525) [inline tests/test_complex.py]: Should be using hwtypes.FPVector

**Comment by rdaly525** [comment]
Thanks for adding in the error bounds. Ill merge this to 'float-test', but we should use hwtypes.FPVector and hwtypes.BitVector in order to construct floats and translate to and from bitvectors.

**Comment by nikhilbhagdikar** [comment]
Sure. Lets clean up the tests post tapeout.

**Pull request merged**